Library/Homebrew/cask/lib/hbc/installer.rb

```diff
@@ -225,7 +225,12 @@ def print_caveats
   def enable_accessibility_access
     return unless @cask.accessibility_access
     ohai "Enabling accessibility access"
-    if MacOS.version <= :mountain_lion
+    if MacOS.version >= :sierra
```

Comment:
Not sure why “it is not safe” on `<= Mountain Lion`, but I think as it is worded now, it can be used for both.